anchore_engine/services/policy_engine/engine/feeds/grypedb_sync.py

```diff
@@ -136,20 +136,22 @@ def _update_grypedb(
         try:
             if grypedb_file_path:
                 GrypeWrapperSingleton.get_instance().init_grype_db_engine(
-                    grypedb_file_path, active_grypedb.checksum
+                    grypedb_file_path, active_grypedb.archive_checksum
```

Comment:
This call will change slightlyt post https://github.com/anchore/anchore-engine/pull/1048 to include the db schema version

Reply:
will rebase after 1048 merge